tests/utils.ts

```diff
@@ -19,3 +22,16 @@ export const fixtureDriftHelper = (received: any, expected: any, pathToFixture:s
     }
   }
 };
+
+export const getPublicKey = (document: DocumentModel, keyId: string): PublicKeyModel | undefined => {
```

Comment:
I think it's good that we remove/refactor this `utils.ts` file sooner or later so let's not put more code in here. How about just move the `Document.ts` file under a `tests/utils/` folder?